README.md

```diff
@@ -26,11 +26,20 @@ Virtual Environment setup:
 Running møte:
  - Start `memcached` by typing `memcached` in the terminal and pressing Return (Enter) key.
  - Edit [config.py](./files/config.py) with necessary information and place the `config.py` in [`mote/`](./mote)
- - Modify the copied `config.py` to use memcached by setting `use_memcached = TRUE`
+ - Modify the copied `config.py` to use memcached by setting `use_memcached = TRUE` (Optional)
    - You can do it by running the following command in command line too: `sed -i '/use_memcached = / s/False/True/' mote/config.py`
  - Enter the Pipenv's newly created virtualenv's shell with `pipenv shell`
  - Run `python runmote.py`
 
+OR
+
+Using `docker-compose` to run møte:
+ - Start `memcached` by typing `memcached` in the terminal and pressing Return (Enter) key.
+ - Edit [config.py](./files/config.py) with necessary information and place the `config.py` in [`mote/`](./mote)
+ - Modify the copied `config.py` to use memcached by setting `use_memcached = TRUE` (Optional)
```

Comment:
Removal of this line, as we're moving forward with file based storage.

Reply:
Done.